app/screens/thread/thread.js

```diff
@@ -148,6 +148,7 @@ export default class Thread extends PureComponent {
                     currentUserId={myMember && myMember.user_id}
                     lastViewedAt={this.state.lastViewedAt}
                     navigator={navigator}
+                    location='thread'
```

Comment:
can we make this a constant somewhere?

Reply:
Thanks, created new file as `app/constants/screen.js`